applications/opentelemetry-operator/0.93.0/helmrelease/opentelemetry.yaml

```
@@ -0,0 +1,38 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: ghcr.io-opentelemetry-operator
```

there is no reason to put ghcr.io in the name - the artifact could be coming from a local harbor registry as well.
```suggestion
  name: ${releaseName}-operator
```

---

done